server/index.js

```diff
@@ -0,0 +1,34 @@
+require('dotenv/config')
+const express = require('express')
+const bodyParser = require('body-parser')
+const { MongoClient } = require('mongodb')
+const app = express()
+
+const port = process.env.PORT || 3000
+
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: true }))
+app.use('/public', express.static(__dirname + '/public'))
```

Comment:
Instead of forcing users to visit `website.com/public`, just omit the first argument to `app.use` so they can go to `website.com`.